org.kde.kcharselect.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.kcharselect",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kcharselect",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "kcharselect",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/22.04.0/src/kcharselect-22.04.0.tar.xz",
+                    "sha256": "e1f86d5d1eae7433ef49339d855caa6f9d847460b29f096c634d786640e261ab"
```

Comment:
can you add the magic that will make it auto-update when there's a new release? See other Gear stuff like https://github.com/flathub/org.kde.okular/blob/master/org.kde.okular.json#L487